### PR TITLE
n2n communication

### PR DESCRIPTION
**Pull request by bartvanb**
First functional version where algorithms can connect to one another

### CODE REVIEW

File: vantage6/server/resource/vpn.py [hunk 1]
```diff
@@ -86,6 +93,68 @@ def get(self):
...
+        if not self._is_server_configured():
+            return {'msg': 'This server does not support VPN'}, \
+                HTTPStatus.NOT_IMPLEMENTED
+
+        # retrieve user based on email or username
+        body = request.get_json()
+        vpn_config = body.get("vpn_config")
+        if not vpn_config:
+            return {"msg": "vpn_config is missing!"}, \
+                HTTPStatus.BAD_REQUEST
```
Comment (frankcorneliusmartin) [inline vantage6/server/resource/vpn.py:136]: Deze `400` mist in de docsting hierboven

File: vantage6/server/resource/vpn.py [hunk 2]
```diff
@@ -86,6 +93,68 @@ def get(self):
...
+                description: Unauthorized or missing permission
+            501:
+                description: The vantage6-server has no VPN service
+            503:
+                description: The VPN server is not reachable
+
+        security:
+            - bearerAuth: []
+
+        tags: ["Other"]
```
Comment (frankcorneliusmartin) [inline vantage6/server/resource/vpn.py]: I think it is good to group all VPN endpoints in "VPN"

**Review by frankcorneliusmartin — CHANGES_REQUESTED** [review]
Also there seems to be a failing unittest

File: vantage6/server/resource/vpn.py (hunk 1 above)
Comment (bartvanb) [reply vantage6/server/resource/vpn.py:136]: Added

File: vantage6/server/resource/vpn.py (hunk 2 above)
Comment (bartvanb) [reply vantage6/server/resource/vpn.py]: Good point. Changed tags for both endpoints to VPN

**Review by frankcorneliusmartin — APPROVED** [review]

**Pull request merged**